src/actions/authActions.js

```diff
@@ -1,16 +1,24 @@
+import config from 'config'
+//import appInsights from 'src/appInsights'
+import appInsights from '../appInsights'
+
 export const LOGIN_IN_PROGRESS = 'LOGIN_IN_PROGRESS'
 export const USER_LOGGED_IN = 'USER_LOGGED_IN'
 export const USER_LOGGED_OUT = 'USER_LOGGED_OUT'
 export const USER_LOGIN_ERROR = 'USER_LOGIN_ERROR'
 
+
 export const loginInProgress = () => ({
   type: LOGIN_IN_PROGRESS
 })
 
-export const userLoggedIn = (user) => ({
-  type: USER_LOGGED_IN,
-  user
-})
+export const userLoggedIn = (user) =>{
+  appInsights.setAuthenticatedUserContext(user.userName.replace(/[,;=| ]+/g, "_"), null, true);
```

Comment:
is this the right place for this code? consider moving to the reducer. consult with Philip what is the right approach.